server/src/routes/map.ts

```diff
@@ -0,0 +1,32 @@
+import { Router } from "express"
+import MapController from "../controllers/map"
+import { FullMap } from "../types"
+
+const mapRouter = Router()
+
+mapRouter.get("/", async (req, res) => {
+  try {
+    const today = new Date()
+    console.log(today)
+    const mapExists: FullMap | null = await MapController.getMap(today)
+
+    if (mapExists) {
```

Comment:
So what's happening if the map doesn't exist for a day? Shouldn't we return the status `404`?

Reply:
That solution should work for now

Reply:
We need a 404 if we request a map that doesn't exist. There isn't a reason we shouldn't be handling it